content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,144 @@ Running this example will fail because it requires that you have [set up remote
 cloud credentials] in DVC Studio to where the model is stored.
 
 </admon>
+</toggle>
+
+## Create new project
```

Comment:
I think we should start splitting this into proper subpages, like we don in DVC and DVCLive cc @tapadipti

Reply:
Created https://github.com/iterative/dvc.org/pull/5124